security/security_tools.md

```diff
@@ -0,0 +1,179 @@
+Report on Dynamic and Static Analysis Security Tools
+====================================================
+
+Recommendations for the CI pipeline
+-----------------------------------
+
+We recommend the following tools for integration in CI for PUL projects:
+
+-   CodeQL
+-   Semgrep
+-   Wapiti
+-   Dastardly
+-   Bearer
+
+Each tool has its own pros and cons, which are detailed in the report below.
+Each project need not implement all of them, most projects would benefit from
+adding one or two to their CI pipelines.
+
+Most of these tools are general: they search for various classes of
+vulnerabilities.  Adding them to your CI pipeline won't provide comprehensive
+protection from any vulnerability, but they will help to:
+* Educate the team about vulnerabilities and mitigation techniques they aren't yet familiar with
+* Enforce secure code patterns
+* Shift security thinking left, from remediation to prevention
+
+The following table gives a brief roundup of the pros and cons to help
+teams decide which tool(s) to start investigating for a particular project:
+
+| Tool | Category | Use case(s) |
+|----|----|----|
+| CodeQL | SAST | A project in Ruby, Java, Javascript, or Python that is looking for a low-barrier setup |
```

Comment:
The table lists SAST first, which I think makes sense as they have the lowest barrier to entry for adoption. But the document below lists Dynamic first. Maybe swap those categories so Static are first?

Also, the SAST and DAST acronyms aren't expanded anywhere, and probably should be.

Reply:
Okay maybe the table orders by recommended use and the sections are all alphabetical? That makes sense, it might just be nice to say this somewhere, I got a little lost trying to go back and forth.